src/OpenTelemetry.Contrib.Instrumentation.AWS/OpenTelemetry.Contrib.Instrumentation.AWS.csproj

```diff
@@ -8,12 +8,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.101.15" />
```

Comment:
I don't think we should upgrade this version. Wouldn't that raise the minimum required version?

Reply:
What do you mean ? It's not an upgrade. We introduced this reference the 1st time.

Reply:
I returned direct ref. to AWSSDK.Core 3.5.1.24 and downgraded AWSSDK.SimpleNotificationService and AWSSDK.SQS to the highest compatible with this version of AWSSDK.Core.